hack/create-custom-cloud-provider-config.sh

```diff
@@ -19,9 +19,8 @@ set -o pipefail
 set +o xtrace
 
 # Install kubectl
-REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
```

Comment:
We want to have `REPO_ROOT` added back into the shell scripts because `REPO_ROOT` stores the script invocation path which could be used later in the shell script. 
_Sourcing_ `REPO_ROOT` breaks the shell script when invoked from other directories.

Reply:
K I am fixing it.